.github/workflows/e2e.yml

```diff
@@ -49,34 +64,18 @@ jobs:
             fail-fast: false
             matrix:
                 specs: ${{ fromJson(needs.cypress_prep.outputs.specs) }}
-
-        services:
-            postgres:
-                image: postgres:12
-                env:
-                    POSTGRES_USER: postgres
-                    POSTGRES_PASSWORD: postgres
-                    POSTGRES_DB: postgres
-                ports:
-                    # Maps port 5432 on service container to the host
-                    # Needed because `postgres` host is not discoverable for some reason
-                    - 5432:5432
-                options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
-            redis:
-                image: redis
-                ports:
-                    # Maps port 6379 on service container to the host
-                    # Needed because `redis` host is not discoverable for some reason
-                    - 6379:6379
-                options: >-
-                    --health-cmd "redis-cli ping"
-                    --health-interval 10s
-                    --health-timeout 5s
-                    --health-retries 5
         steps:
             - name: Checkout
               uses: actions/checkout@v2
 
+            - name: Start stack with Docker Compose
+              run: |
+                  docker-compose -f ee/docker-compose.ch.yml down
+                  docker-compose -f ee/docker-compose.ch.yml up -d db clickhouse zookeeper kafka redis &
+
+            - name: Add kafka host to /etc/hosts for kafka connectivity
+              run: sudo echo "127.0.0.1 kafka" | sudo tee -a /etc/hosts
```

Comment:
non-blocking

What's the need for this? Could you not just set KAFKA_URL=kafka=://localhost:9092 ?

Reply:
I copied this from e2e.yml but did try to run it without this: https://github.com/PostHog/posthog/runs/4691498447?check_suite_focus=true. Looks like we hardcode kafka in some places. Maybe worth fixing but not super high priority.